src/parser/jobs/rpr/modules/EnhancedGibbetGallows.tsx

```diff
@@ -0,0 +1,117 @@
+			return <Trans id="rpr.enhanced-gibbet-gallows.expired.why">
+				<Plural value={expiredGibbet} one="# Enhanced Gibbet proc" other="# Enhanced Gibbet procs" /> and <Plural value={expiredGallows} one="# Enhanced Gallows proc" other="# Enhanced Gallows procs" /> were dropped, causing a loss of {(expiredGibbet + expiredGallows) * PROC_POTENCY} potency.
+			</Trans>
+		}
+		if (expiredGibbet < 0) {
+			return <Trans id="rpr.enhanced-gibbet-gallows.expired-gibbet.why">
+				<Plural value={expiredGibbet} one="# Enhanced Gibbet proc was" other="# Enhanced Gibbet procs were" /> dropped causing a loss of {expiredGibbet * PROC_POTENCY} potency
+			</Trans>
+		}
+		return <Trans id="rpr.enhanced-gibbet-gallows.expired-gallows.why">
+			<Plural value={expiredGallows} one="# Enhanced Gallows proc was" other="# Enhanced Gibbet procs were" /> dropped causing a loss of {expiredGallows*PROC_POTENCY} potency
+		</Trans>
+	}
+	private getOverwrittenProcContent(overwrittenGibbet: number, overwrittenGallows: number) {
+		if (overwrittenGibbet > 0 && overwrittenGallows > 0) {
+			return <Trans id="rpr.enhanced-gibbet-gallows.overwritten.content">
+				Avoid using <DataLink action="GALLOWS" /> when you have <DataLink status="ENHANCED_GIBBET" /> or <DataLink action="GALLOWS" /> when you have <DataLink status="ENHANCED_GIBBET" />
+			</Trans>
+		}
+		if (overwrittenGibbet > 0) {
+			return <Trans id="rpr.enhanced-gibbet-gallows.overwritten-gibbet.content">
+				Avoid using <DataLink action="GALLOWS" /> when you have <DataLink status="ENHANCED_GIBBET" />
+			</Trans>
+		}
+		return <Trans id="rpr.enhanced-gibbet-gallows.overwritten-gallows.content">
+			Avoid using <DataLink action="GIBBET" /> when you have <DataLink status="ENHANCED_GALLOWS" />
+		</Trans>
+	}
+	private getOverwrittenProcWhy(overwrittenGibbet: number, overwrittenGallows: number) {
+		if (overwrittenGibbet > 0 && overwrittenGallows > 0) {
+			return <Trans id="rpr.enhanced-gibbet-gallows.overwritten.why">
+				<Plural value={overwrittenGibbet} one="# Enhanced Gibbet proc" other="# Enhanced Gibbet procs" /> and <Plural value={overwrittenGallows} one="# Enhanced Gallows proc" other="# Enhanced Gallows Procs" /> were lost due to being overwritten, causing a loss of {(overwrittenGibbet + overwrittenGallows) * PROC_POTENCY} potency.
+			</Trans>
+		}
+		if (overwrittenGibbet > 0) {
+			return <Trans id="rpr.enhanced-gibbet-gallows.overwritten-gibbet.why">
+				<Plural value={overwrittenGibbet} one="# Enhanced Gibbet proc was" other="# Enhanced Gibbet procs were" /> lost due to being overwritten, causing a loss of {overwrittenGibbet * PROC_POTENCY} potency
+			</Trans>
+		}
+		return <Trans id="rpr.enhanced-gibbet-gallows.overwritten-gallows.why">
+			<Plural value={overwrittenGibbet} one="# Enhanced Gallows proc was" other="# Enhanced Gallows procs were" /> lost due to being overwritten, causing a loss of {overwrittenGallows * PROC_POTENCY} potency
+		</Trans>
+	}
```

Comment:
Can we simplify this, and maybe get some spacing in there so it's easier to read?

Reply:
would just "Avoid overwriting your procs" / "Avoid letting your procs expire" be sufficient?

Reply:
I think there's 2 options, either we single suggestion with the text you've just described, or we rename this module to just `Procs` or something, add Void/Cross Reaping, and have separate suggestions for each skill, which tbh I think is a better approach, cleaner and clearer even if it's 8 suggestions potentially.

Reply:
I like the second idea slightly better. I'll handle the refactor and logic changes tomorrow before raid.

Thanks for all your help and guidance!